test/test_persistent.rb

```diff
@@ -156,7 +157,7 @@ def test_one_with_keep_alive_header
   end
 
   def test_persistent_timeout
-    @server.persistent_timeout = 1
+    @server.instance_variable_set(:@persistent_timeout, 1)
```

Comment:
Later, we should rewrite the tests so we can pass this in as an option when we create the server object, rather than have to change the instance variable.